src/visualization/visualize-donor-data-scatterplot.py

```diff
@@ -0,0 +1,163 @@
+# Import Packages
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+
+import pandas as pd
+import numpy as np
+
+# Functions
+def get_age_category(value):
+    if pd.isnull(value):
+        return np.nan
+    elif value < 7:
+        return 1
+    elif value < 14:
+        return 2
+    elif value < 25:
+        return 3
+    elif value < 50:
+        return 4
+    else:
+        return 5
+
+
+def get_age_label(value):
+    if value == 1:
+        return "0 - 7"
+    elif value == 2:
+        return "7 - 14"
+    elif value == 3:
+        return "14 - 25"
+    elif value == 4:
+        return "25 - 50"
+    elif value == 5:
+        return "> 50 "
+    else:
+        return np.NaN
+
+
+def read_and_format_data(file_path):
+    df = pd.read_csv(file_path)
+
+    df['Age Category'] = df['age'].apply(get_age_category)
+
+    df = df[df['category'] == "age-ylw"]
+
+    df = df.melt(id_vars=["Age Category", "age"],
+                 var_name="Indicator Name",
+                 value_name="Value")
+
+    return df
+
+
+# Read and Format Data File
+file_path = '/Users/anneevered/Desktop/2019-07-17-aggregate-cgm-stats.csv.gz'
```

Comment:
I know this is exploratory work, so totally fine to have this in here, but in general, it is not good practice to hard code paths to your local machine. I think it would be great if this could pull from a google drive folder, or in the interim you could prompt the user to enter the path.